### ISSUE TITLE
Error('unsupported locale setting')

### ISSUE THREAD

**Issue by lalalaurenv** [CS-4820-Library-Project/COUNTER-5-Report-Tool#225]
When following the ReadMe section '[Setup Instructions (Windows)](https://github.com/CS-4820-Library-Project/COUNTER-5-Report-Tool#setup-instructions-windows)', the following error occurs at the step 'Type: python maindriver.py'

(<class 'locale.Error'>, Error('unsupported locale setting'), <traceback object at 0x000002582B752440>)

Running Windows 10. No other errors up to this point.

**Comment by mbelvadi** [comment]
Hi. if you are in a non-English locale, we recommend you set:
 LC_TIME=en_US.UTF-8
as the application was entirely written in English and does not support other languages at this time.
I will add the idea of adding this directly into the code in our future fixes planning.

**Comment by lalalaurenv** [comment]
Thanks for this! I am on an English computer (all region and language settings set to English) but there must be something not quite right with it.

Any tips for where to enter that command?

Entering it in command line results in: "'LC_TIME' is not recognized as an internal or external command, operable program or batch file."

For what it's worth, I get ('English_Canada', '1252') when entering the following in Python:
>import locale
>locale.getlocale()

**Comment by mbelvadi** [comment]
I just checked with my lead programmer and so far he thinks the code to handle this is already in the tool, so we're not quite sure what's going on but it may be related to your python configuration. Since you're on Windows 10, have you tried just using the pre-compiled EXE for windows that we provided?  That should have everything needed all bundled and ready to run, not dependent on the settings in your own python config at all.

**Comment by sesan07** [comment]
If you're willing to edit the file, you can open MainDriver.py and edit line 40.

replace 
locale.setlocale(locale.LC_ALL, 'en_US.UTF-8')
with
locale.setlocale(locale.LC_ALL, 'en_CA.UTF-8')

Save the file and try running python MainDriver.py again.

**Comment by lalalaurenv** [comment]
Thanks to both of you. I have tried the pre-compiled [Windows .exe](https://github.com/CS-4820-Library-Project/COUNTER-5-Report-Tool/releases), and unfortunately it just opens a very quick terminal window and then disappears with no further results.

I just edited maindriver.py as suggested and changed the en_US bit to en_CA, but still the same error upon running from the command line 'python maindriver.py':
>(<class 'locale.Error'>, Error('unsupported locale setting'), <traceback object at 0x000002860BA92280>)

Puzzling!

**Comment by sesan07** [comment]
Locale values seem to differ between operating sytems. Since you're on windows can you try replacing 'en_US.UTF-8' with 'English.utf8'

**Comment by lalalaurenv** [comment]
Here's how that line reads in maindrive.py now:
>locale.setlocale(locale.LC_ALL, 'English.utf8')

Still throwing the same error upon trying 'python maindrive.py' from the command line, unfortunately. I tried 'English.utf-8' as well in case the hyphen made a difference, but no luck.

**Comment by sesan07** [comment]
Yeah, that's really weird. The only thing that I can suggest right now is to remove that line completely.
